### PR TITLE
Show metadata.labels in service-catalog ui

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Show `metadata.labels` in main service-catalog ui,
- Add two additional categories in filter dropdown: `Basic Filter` for `metadata.labels.showcase` and/or `metadata.labels.local` and `Connected Applications` for `metadata.labels.connected-app`, 
- Add possibility to filter classes by `metada.labels` (`local`, `showcase`, `connected-app`),
- Improve `Tooltip` component in `@kyma-project/react-components` (by default tooltip with white text on black background),
- Update OSBA contract.

**Related issue(s)**
See also #127

### CODE REVIEW

File: catalog/OSBAContract.md [hunk 1]
```diff
@@ -4,13 +4,16 @@ Catalog-ui directly uses the [ui-api-layer](https://github.com/kyma-project/kyma
...
+| Number | OSBA field                    | Fallbacks  | Description                                                                                                                |
+| ------ | ----------------------------- | ---------- | -------------------------------------------------------------------------------------------------------------------------- |
+| (1)    | metadata.displayName          | name*, id* | If **metadata.displayName, name, id** fields are not present, the given Service Class does not appear on the landing page. |
+| (2)    | metadata.providerDisplayName  | -          | If not provided, UI displays without this information.                                                                     |
+| (3)    | description\*                 | -          | If not provided, UI displays without this information.                                                                     |
+| (4)    | metadata.labels               | -          | If not provided, UI displays without label `connected-app`.                                                                |
```
Comment (kazydek) [inline catalog/OSBAContract.md]: ` If not provided, UI does not display any labels.`

File: catalog/OSBAContract.md [hunk 2]
```diff
@@ -4,13 +4,16 @@ Catalog-ui directly uses the [ui-api-layer](https://github.com/kyma-project/kyma
...
+| (5)    | metadata.labels.local and/or metadata.labels.showcase | - | If not provided, filtering by Labels `local` or `showcase` are not possible.                                |
```
Comment (kazydek) [inline catalog/OSBAContract.md]: ``filtering by Labels `local` or `showcase` are not possible.  `` -> ``filtering by the `local` or `showcase` labels is not possible.  ``

File: catalog/OSBAContract.md [hunk 3]
```diff
@@ -4,13 +4,16 @@ Catalog-ui directly uses the [ui-api-layer](https://github.com/kyma-project/kyma
...
+| (6)    | tags                          | -          | If not provided, filtering by Tag is not possible.                                                                         |
+| (7)    | metadata.labels.connected-app | -          | If not provided, filtering by Label `connected-app` is not possible.                                                       |
```
Comment (kazydek) [inline catalog/OSBAContract.md]: ``by Label `connected-app` `` -> ``by the `connected-app` label``

File: catalog/src/components/ServiceClassList/Cards/Card.component.js [hunk 4]
```diff
@@ -13,16 +13,26 @@ import {
...
+const Card = ({ title, company, description, imageUrl, labels, onClick }) => {
   const itemId = title
     ? title
         .split(' ')
         .join('-')
         .toLowerCase()
     : '';
 
+  const labelsDescription = {
+    local: 'Class will provision physical resources inside cluster',
```
Comment (kazydek) [inline catalog/src/components/ServiceClassList/Cards/Card.component.js]: `This Service Class provisions physical resources inside the cluster.`

File: catalog/src/components/ServiceClassList/Cards/Card.component.js [hunk 5]
```diff
@@ -13,16 +13,26 @@ import {
...
+    showcase:
+      'Nothing to use on a production but rather to showcase some specific functionality',
```
Comment (kazydek) [inline catalog/src/components/ServiceClassList/Cards/Card.component.js]: `Provision this Service Class only to demonstrate a specific functionality. Do not use it on the production cluster. `

**Comment by derberg** [comment]
@pkosiec not sure really what you mean, but I'm sure I want:
- to see it in action before we merge
- to have it clear in documentation that metadata.label is not really a part of OSBA, this is actually a first time we introduced a custom attribute in the metadata object that is not part of core OSBA and the standard convention

File: catalog/src/commons/helpers.js [hunk 6]
```diff
@@ -152,3 +152,7 @@ export function randomNameGenerator() {
     nouns[getRandomInt(0, nouns.length + 1)]
   ).toLowerCase();
 }
+
+export function isTrueSet(value) {
```
Comment (pkosiec) [inline catalog/src/commons/helpers.js]: `isStringValueEqualToTrue` would be more verbose I think

Comment (magicmatatjahu) [reply catalog/src/commons/helpers.js]: Done...

**Pull request merged**